skyportal/handlers/api/followup_request.py

```diff
@@ -3,7 +3,7 @@
 
 from baselayer.app.access import auth_or_token
 from ..base import BaseHandler
-from ...models import (
+from ...models import (  # noqa
```

Comment:
Isn't this already in master?

Reply:
Not according to github

Reply:
What's being imported here that's not being used?

Reply:
We should just remove the offending import rather than sweep it under the rug with the noqa comment. We use that sparingly in models.py for instance where we import things that aren't explicitly used there because other modules need to import them from models.py